x-pack/plugins/ml/public/application/components/job_selector/job_selector_table/job_selector_table.js

```diff
@@ -47,7 +47,10 @@ export function JobSelectorTable({
 
   const {
     services: {
-      application: { navigateToApp },
+      application: {
+        navigateToApp,
+        capabilities: { ml: mlCapabilities },
```

Comment:
As discussed, to close off https://github.com/elastic/kibana/issues/163575 we also need to add a check for ML nodes being available.